### DISCUSSION TITLE
Is it possible to add 2 or more hyperlinks in one cell?

### DISCUSSION THREAD

**Discussion by gu3nter** [exceljs/exceljs#2622] [Q&A]
Hello,

hopefully you can help me, many thanks to the great exceljs.

At the moment i´m only able to add a cell one string with one hyperlink.
So the question is:
Is it possible to add 2 or more hyperlinks in one cell for parts of the string?

My current code for e.g cell A1:

row = worksheet.getRow(1);
row.getCell(1).value = {text: 'ExampleString', hyperlink: 'https://example.com', tooltip: 'www.example.com'};

So i want to have two hyperlinks in cell A1. one for 'Example' and a second for 'String'.

Regards,

René

**Comment by wissam-natex** [comment]
+1

**Comment by karan-cswg** [comment]
+1

**Comment by vucong2018** [comment]
+1
